analyzer/analyzer.go

```diff
@@ -2,6 +2,7 @@ package analyzer
 
 import (
 	"fmt"
+	"github.com/rodrigo-brito/gocity/lib/file"
```

Comment:
please, reorder the import, keep it together with  `gocity/lib` import.

Reply:
Hi, did you mean to tag @HemantJoshi11 , for these changes? because this pull request was made by them.

Reply:
Hi @vidu171, sorry for the mistake.